examples/Project.toml

```diff
@@ -18,12 +15,16 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
 OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
+ProgressLogging = "33c8b6b6-d38a-422a-b730-caa89a2f386c"
```

Comment:
where do we need this?

Reply:
This is used for progress bars in `inertial_gravity_wave.jl`. Generating the analytical solution takes a very long time, so I added `@progress` to some for loops.